gradle.properties

```diff
@@ -1,5 +1,5 @@
 group = com.cdancy
-version = 0.0.10-pjr
+version = 0.0.10-SNAPSHOT
```

Comment:
In fact, it should actually be `0.0.11-SNAPSHOT`, since version 0.0.10 is already released, and you targeted this PR to v0.0.11 milestone.